src/cmap/auth/mongodb_oidc/callback_workflow.ts

```diff
@@ -278,19 +166,3 @@ function isCallbackResultInvalid(tokenResult: unknown): boolean {
   if (!('accessToken' in tokenResult)) return true;
   return !Object.getOwnPropertyNames(tokenResult).every(prop => RESULT_PROPERTIES.includes(prop));
```

Comment:
```suggestion
  return !Object.keys(tokenResult).every(prop => RESULT_PROPERTIES.includes(prop));
```

It's probably stricter than necessary to enforce that the callback returns an object with own properties.

Reply:
@baileympearson I think there might be a small misunderstanding here – `Object.keys()` also only returns the names of own properties, but excludes non-enumerable ones

(And I have no strong feelings about _that_ distinction, tbh. A callback should typically just return a good-old regular JS object most of the time anyway.)